webapp/publisher/snaps/views.py

```diff
@@ -496,11 +500,11 @@ def post_register_name_dispute():
 @publisher_snaps.route("/request-reserved-name")
 @login_required
 def get_request_reserved_name():
-    user = publisher_api.get_account(flask.session)
+    stores = admin_api.get_stores(flask.session)
```

Comment:
The account no longer has the list of stores, so get the list of stores directly from the API